pkg/generators/forwarding/fluentd/fluent_conf_test.go

```diff
@@ -950,11 +950,11 @@ var _ = Describe("Generating fluentd config", func() {
 						ca_file '/var/run/ocp-collector/secrets/my-infra-secret/ca-bundle.crt'
 						type_name _doc
 						write_operation create
-						reload_connections "#{ENV['ES_RELOAD_CONNECTIONS'] || 'true'}"
+						reload_connections 'true'
 						# https://github.com/uken/fluent-plugin-elasticsearch#reload-after
-						reload_after "#{ENV['ES_RELOAD_AFTER'] || '200'}"
+						reload_after '200'
 						# https://github.com/uken/fluent-plugin-elasticsearch#sniffer-class-name
-						sniffer_class_name "#{ENV['ES_SNIFFER_CLASS_NAME'] || 'Fluent::Plugin::ElasticsearchSimpleSniffer'}"
+						
```

Comment:
Is the `sniffer_class_name` drop intentionally?

Reply:
yes.  this should be the default.  I believe this is here for when the change was not part of the original plugin

Reply:
@periklis looks as I was wrong and should actually check the default https://bugzilla.redhat.com/show_bug.cgi?id=1864397  https://github.com/uken/fluent-plugin-elasticsearch#sniffer-class-name